karma.conf.js

```diff
@@ -0,0 +1,82 @@
+// Karma configuration
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+    plugins: [
+      'karma-coverage',
+      'karma-chrome-launcher',
+      'karma-jasmine'
+    ],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'app/public.js',
+      {pattern: 'test/**/*.png', included: false},
```

Comment:
That means we're making those files available in the browser but not automatically loading them as it fails with 

> Refused to execute script from 'http://localhost:9876/base/test/assets/placeholder-100x80.png' because its MIME type ('image/png') is not executable.